packages/compiler/src/core/checker.ts

```diff
@@ -1454,7 +1454,7 @@ export function createChecker(program: Program): Checker {
             createDiagnostic({
               code: "intersect-duplicate-property",
               format: { propName: prop.name },
-              target: node,
+              target: prop,
```

Comment:
how does it make it better for the trait issue?

This seems worse to me now,  it shows the error somewhere far from usage.

https://cadlplayground.z22.web.core.windows.net/prs/2299/?c=bW9kZWwgQSB7CiAgZm9vOiBzdHJpbmc7Cn0KCsYcQtUcCmFsaWFzIEMgPSBBICYgQjs%3D&e=%40typespec%2Fopenapi3&options=%7B%7D


I think the solve the trait issue it need the underlying stacktrace of diagnostic so it can report the error higher up

Reply:
Right now it just displays the error on the import statement, which is the absolute worse. I can't find the underlying issue for that but I thought we had one. This was just an experiment to get the playground up so I could see the difference. If there's no easy fix, I'll just have to defer until that larger issue is fixed.

Reply:
It shows at the top in the playground because it reports the error in another file that the main.tsp 
If you run this code in the command line you get this 
<img width="1523" alt="image" src="https://github.com/microsoft/typespec/assets/1031227/6c1ebde5-90db-4830-a9fc-c17a14d14690">

Issue in question to solve those https://github.com/microsoft/typespec/issues/1321

Reply:
Ah, perfect. In that case, I'll close this and leave that issue to address the stack trace.